src/test/java/school/lemon/changerequest/java/junit/hw1/task1/api/ContactApiTest.java

```diff
@@ -0,0 +1,215 @@
+package school.lemon.changerequest.java.junit.hw1.task1.api;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import school.lemon.changerequest.java.junit.hw1.task1.model.*;
+
+import static org.apache.commons.lang3.RandomStringUtils.randomNumeric;
+import static org.junit.Assert.*;
+
+public class ContactApiTest {
+
+    ContactApi contactApi = null;
+    Contact testContact = null;
+
+    @Before
+    public void setUp() {
+        contactApi = new ContactApiImpl(new ContactIdGeneratorImpl());
+        testContact = new Contact((long) 0, new CallNumber("+1", "2345", "6789012"), "Alex", "Marukhnenko");
+    }
+
+    @Test
+    public void findTest() {
+        contactApi.create(testContact);
+        Contact foundContact = contactApi.find(new CallNumber("+1", "2345", "6789012"));
+        assertEquals(testContact, foundContact);
+    }
+
+    @Test(expected = ContactNotFoundException.class)
+    public void findNotExistedContactTest() {
+        contactApi.find(new CallNumber("a", "bcde", "fghi"));
+    }
+
+    @Test(expected = NullPointerException.class)
+    public void findNullTest() {
+        contactApi.find(null);
+    }
+
+
+    @Test
+    public void getTest() {
+        Contact[] list = new Contact[3];
+        for (int i = 0; i < 3; i++)
+            list[i] = contactApi.get((long) (1000 + i));
+
+        initContactsTest(list);
```

Comment:
If you use some test-helper methods — put them just after the test case, so that you won't need to scroll and look for the method.